infrastructure/validators/operational-tasks/monitor-node.md

```diff
@@ -0,0 +1,371 @@
+![6-dashboard-metric](/images/infrastructure/validator/operational-tasks/6-dashboard-metric.webp)
+
+## Install and Configure Optional `Alertmanager`
+
+You can also configure the optional `Alertmanager` to help predict potential problems or notify you of an active problem with your server. You can receive alerts via Slack, email, and Matrix among other messaging services. You can configure email notifications to alert you if your node goes down. The following sections cover how to install and configure the `AlertManager`.
+
+First, download the latest binary of `AlertManager` and unzip it by running the following command:
+```bash
+--8<-- 'code/infrastructure/validators/operational-tasks/monitor-node/unzip-alert-manager.md'
+```
+
+### Setup Gmail
+
+To use Gmail, you must generate an `app password` in your Gmail account to allow `AlertManager` to send you alert notification emails. Visit Google support for more details on setting up your [`app password`](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank}. 
+
+Once you complete the setup, you should see something like the following:
+
+![grafana-am-1](/images/infrastructure/validator/operational-tasks/1-alert-manager.webp)
+
+Be sure to backup this password in a secure location.
+
+### Configure AlertManager
+
+There is a configuration file named `alertmanager.yml` inside the directory that you unzipped in the first step, but that isn't of use. Create a custom `alertmanager.yml` file under `/etc/alertmanager` with the following configuration:
+
+??? tip "Change `alertmanager` ownership"
+    Ensure to change the ownership of `/etc/alertmanager` to `prometheus` by executing the following command:
+    ```bash
+    sudo chown -R prometheus:prometheus /etc/alertmanager
+    ```
+
+```
+--8<-- 'code/infrastructure/validators/operational-tasks/monitor-node/alert-manager.yml'
+```
+
+With the preceding configuration, alerts will be sent using the email you set preceding. Remember to replace `YOUR_EMAIL` with the email where you want to receive alerts and paste the app password you saved earlier to the `YOUR_APP_PASSWORD`.
+
+Next, create another `systemd` configuration file named `alertmanager.service` by running the following command:
+``` bash
+sudo nano /etc/systemd/system/alertmanager.service 
+```
+Add the following code to the configuration file you just created:
+
+???tip  "SERVER_IP"
+    Change to your host IP address and make sure port 9093 is opened
+
+``` bash
+--8<-- 'code/infrastructure/validators/operational-tasks/monitor-node/systemd-alert-config.md'
+```
+
+To start the `Alertmanager`, run the following commands:
+
+```
+sudo systemctl daemon-reload && sudo systemctl enable alertmanager && sudo systemctl start alertmanager && sudo systemctl status alertmanager
+```
+
+```
+● alertmanager.service - AlertManager Server Service
```

Comment:
Another really old terminal output. Please run through this process and get fresh info. Thank you!